### PR TITLE
Package fixes

### PR DESCRIPTION
**Pull request by RyokoNod**
Added helper function to remove trailing/preceding slashes, fixed dplyr operation in get_mp

### CODE REVIEW

File: R/main_script.R [hunk 1]
```diff
@@ -1,3 +1,13 @@
...
+#' @param input_string input string
+remove_trailing_and_leading_slashes <- function(input_string) {
+  # Remove leading slashes
+  input_string <- sub("^/*", "", input_string)
+  # Remove trailing slashes
+  input_string <- sub("/*$", "", input_string)
+  return(input_string)
+}
+
 #' Call an API once
```
Comment (FrederikHeylen) [inline R/main_script.R:11]: Best practice is to create documentation of a new function with devtools::document().
Also make sure to run devtools::test() to run tests.

Comment (FrederikHeylen) [reply R/main_script.R:11]: Function looks good!

File: R/main_script.R [hunk 2]
```diff
@@ -47,21 +52,21 @@ call_api_once <- function(URL, path = NULL, query, ...) {
 #' @importFrom dplyr %>%
 call_api_multiple_times <- function(iterator, URL, path, query, resultVector, use_parallel = TRUE) {
   message("Making ", length(iterator), " calls.")
-
+
```
Comment (FrederikHeylen) [inline R/main_script.R:55]: What happened to all the indentations?

**Review by FrederikHeylen — COMMENTED** [review]
See comments!

File: R/main_script.R [hunk 3]
```diff
@@ -47,21 +52,21 @@ call_api_once <- function(URL, path = NULL, query, ...) {
...
     doParallel::registerDoParallel(cl)
     on.exit(parallel::stopCluster(cl))
-
+    
     time_used <- system.time({
       list <- foreach::foreach(
         i = seq_along(iterator),
         .packages = c("dplyr", "purrr", "httr", "jsonlite"),
-        .export = c("call_api_once"),
+        .export = c("call_api_once", "remove_trailing_and_leading_slashes"),
```
Comment (FrederikHeylen) [inline R/main_script.R:69]: Really excellent that you found this!

Comment (RyokoNod) [reply R/main_script.R:69]: Courtesy of ChatGPT ;)

File: R/main_script.R (hunk 2 above)
Comment (RyokoNod) [reply R/main_script.R:55]: I am not sure. It looks like my RStudio did it automatically, but I don't see any settings for it

**Pull request merged**